utils/build/docker/nodejs/express4/app.js

```diff
@@ -134,13 +135,64 @@ app.get("/users", (req, res) => {
   }
 });
 
+app.get("/dsm", (req, res) => {
+  const kafka = new Kafka({
+    clientId: 'my-app',
+    brokers: ['kafka:9092'],
+    retry: {
+      initialRetryTime: 100, // Time to wait in milliseconds before the first retry
+      retries: 20, // Number of retries before giving up
+    },
+  })
+  console.log("in endpoint")
```

Comment:
`console.log`s in the sample app are okay but this looks like leftover debugging code.